src/Libraries/DSOffice/Excel.cs

```diff
@@ -416,7 +417,7 @@ internal object[][] Data
         /// </summary>
         /// <param name="wbook"></param>
         /// <param name="sheetName"></param>
-        internal WorkSheet (WorkBook wbook, string sheetName)
+        internal WorkSheet (WorkBook wbook, string sheetName, bool overWrite = true)
```

Comment:
Soon I will be adding a default argument `overwrite` and assign it to `true` in the `WriteToFile` excel node. This implementation will make use of this argument in this `WorkSheet` constructor. If the flag is `true`, the worksheet will be overwritten and if it's `false`, the worksheet will be overwritten partially depending on the data and input row and column info.